handlers/issue.go

```diff
@@ -59,13 +60,47 @@ func (s *Server) handleIssueCommentEvent(body []byte) {
 		if len(labelSlice) > 0 {
 		}
 	}
-
+	
 	if retestReg.MatchString(comment) {
 		// "/retest"
 		s.SendToCI(info)
 	} else if testReg.MatchString(comment) {
 		// TODO: trigger particular job(s)
 		s.SendToCircleCI(body)
+	}*/ 
+	
+	ctx := context.Background()
+
+	list,_,err := client.Repositories.ListCollaborators(ctx,"swx457056","test-ci-bot",nil)
```

Comment:
remove hard-coding of owner and repo

Reply:
okay...but this is not the actual code that we are working on.

Reply:
okay....but this is not the actual code change.This was the initial code changes from Huawei PaaS. can you please review changes made in this commit:https://github.com/Huawei-PaaS/ci-bot/pull/2/commits/a6c19b6d6d011fc30a1748a61d1a2854bff6a56b